### DISCUSSION TITLE
MATCH ADVANCED ORDER

### DISCUSSION THREAD

**Discussion by mustafa222lk** [ProjectOpenSea/seaport-js#164] [General]
Hey there,
I wanna accept a collection bid on opensea using their seaport protocol. I see they are using match advanced orders on opensea but when i try to find information about it. I cant find anything. Its not a function you can call using this libary. I also tried using fulfill but it just doesant work. What is the correct way to accept collection based offers ? I am able to accept nft related offers but when it come to collection based offers that have a criteria type 4-5, i just dont know what to do. Anyone here can help me and explain me what to do ?

**Comment by ryanio** [comment]
Hey @mustafa222lk sorry for the delay, there is more information here https://docs.opensea.io/reference/build-collection-offers and https://docs.opensea.io/reference/fulfill-a-listing

I do think you're right matchAdvancedOrders was never implemented in this library. We're open to PRs if that's something you'd be interested in adding! It would be great to have
